src/mimir_config.py

```diff
@@ -162,7 +162,6 @@ def _build_tls_config(self) -> Dict[str, Any]:
         }
         return {
             "http_tls_config": tls_config,
-            "grpc_tls_config": tls_config,
```

Comment:
Please add a FIXME with link to [issue](https://github.com/canonical/mimir-coordinator-k8s-operator/issues/141).
If we can also keep a blank placeholder, even better:
```
"grpc_tls_config": {
    # FIXME ...
},
```